fake-lmp-device-register

```diff
@@ -23,11 +23,16 @@ class Options(NamedTuple):
     production: bool
 
 
-def create_key(uuid: str, factory: str, production: bool) -> Tuple[bytes, bytes]:
-    key = subprocess.check_output(
-        ["openssl", "ecparam", "-genkey", "-name", "prime256v1"]
+def create_key(uuid: str, factory: str, production: bool, sota_dir: str) -> Tuple[bytes, bytes]:
+    keyfile_path = os.path.join(sota_dir, "pkey.pem")
+
+    subprocess.check_call(
+        ["openssl", "ecparam", "-genkey", "-name", "prime256v1", "-out", keyfile_path]
     )
 
+    with open(keyfile_path, "rb") as keyfile:
+        key_data = keyfile.read()
```

Comment:
I was wondering _why do we need to read this key at all_?

So, I spotted a ghost usage of it in _main_ at line 93:
```
    with open(os.path.join(opts.sota_dir, "pkey.pem"), "wb") as f:
        f.write(pkey)
```
... Which apparently tries to write this same key to this same file.

So, I do recommend to change the `create_key` signature to only return the `csr`, and remove that ghost code (which is not relevant after your change).

Reply:
I review this one and you are right
We can cleanup it. It is no longer required.
Good catcher.